src/icons/logo.tsx

```diff
@@ -0,0 +1,10 @@
+import logo from '@/assets/svg/rss-logo.svg';
+import Image from '@/features/image';
+
+interface LogoProps {
+  type: 'navbar' | 'footer';
+}
+
+export const Logo = ({ type }: LogoProps) => {
+  return <Image className={type !== 'navbar' ? 'white-logo' : ''} src={logo} alt="logo" />;
```

Comment:
Please add `classnames` package, and change to this
https://www.npmjs.com/package/classnames

```tsx
className={classNames({
  'white-logo': type === 'footer',
)}
```

Reply:
Done